src/app/scheme/scheme-routing.module.ts

```diff
@@ -21,25 +21,33 @@ const schemeRoutes: Routes = [{
     canActivate: [AuthGuard],
     children: [{
         path: ':name',
+        data: { title: '%DEVICE%' },
```

Comment:
Почему DEVICE ? Может всё таки SCHEME?
```suggestion
        data: { title: '%SCHEME%' },
```